include/odp/api/spec/timer.h

```diff
@@ -185,6 +185,27 @@ uint64_t odp_timer_ns_to_tick(odp_timer_pool_t timer_pool, uint64_t ns);
  */
 uint64_t odp_timer_current_tick(odp_timer_pool_t timer_pool);
 
+/**
+ * Sample tick values of timer pools
+ *
+ * Reads timer pool tick values simultaneously (or closely back-to-back) from all requested timer
+ * pools, and outputs those on success. Optionally, outputs corresponding source clock (HW) counter
+ * values, which are implementation specific and may be used for debugging. When a timer pool does
+ * not support reading of the source clock value, zero is written instead. Values are written into
+ * both arrays in the same order which timer pools are defined. Nothing is written on failure.
```

Comment:
Perhaps `Values are written into the output arrays in the same order which timer pools were defined.`.

Reply:
Sounds better, fixed in v4.